data/presets/presets/man_made/observatory.json

```diff
@@ -0,0 +1,15 @@
+{
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "type": [
```

Comment:
This should be "terms"

Reply:
By the way this is what is causing the test failures.   "type" doesn't fit here and so the build process will raise an error when it tries to validate the `observatory.json` file against the [preset schema](https://github.com/openstreetmap/iD/blob/master/data/presets/schema/preset.json).